server/conn.go

```diff
@@ -785,6 +785,9 @@ func (cc *clientConn) handleLoadData(goCtx goctx.Context, loadDataInfo *executor
 	}
 
 	txn := loadDataInfo.Ctx.Txn()
+	if err1 := txn.FlushStatement(err == nil); err1 != nil {
```

Comment:
I prefer discard/rollback the StmtTxn explicitly, because use a boolean false to indicate err will be error prone.